### PR TITLE
Deps upgrade

### PR DESCRIPTION
**Pull request by treyhuffine**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,58 +19,62 @@
...
     "emotion-normalize": "^7.0.1",
-    "enzyme": "^3.7.0",
-    "enzyme-adapter-react-16": "^1.6.0",
+    "enzyme": "^3.8.0",
+    "enzyme-adapter-react-16": "^1.9.1",
     "gh-pages": "^2.0.1",
     "lodash": "^4.17.10",
-    "react": "^16.4.2",
-    "react-dom": "^16.4.2",
+    "react": "^16.8.1",
```
Comment (mager) [inline package.json]: 😎

File: package.json [hunk 2]
```diff
@@ -19,58 +19,62 @@
...
+    "build": "npx ts-node ./node_modules/react-static/bin/react-static build",
     "serve": "serve dist -p 3000",
     "predeploy": "yarn build",
     "deploy": "gh-pages -d dist",
-    "deploy:stage": "yarn stage && netlify deploy"
+    "deploy:stage": "yarn stage && netlify deploy --prod"
   },
   "dependencies": {
-    "axios": "^0.16.2",
+    "@reach/router": "^1.2.1",
```
Comment (mager) [inline package.json]: Never heard of https://github.com/reach/router.

File: static.config.tsx [hunk 3]
```diff
@@ -5,13 +5,11 @@ import { buildTeamRoutes } from './src/utils/route';
 import { Meta } from './src/utils/types';
 import { projects } from './data';
 
-const typescriptWebpackPaths = require('./webpack.config.js');
-
-const PRODUCTION_URL = 'https://postmates.com/open-source';
-const STAGING_URL = 'https://awesome-lamarr-c46055.netlify.com';
+const PRODUCTION_URL = 'https://postmates.io';
```
Comment (mager) [inline static.config.tsx]: 👏

**Comment by mager** [comment]
Awesome work!

File: package.json (hunk 2 above)
Comment (treyhuffine) [reply package.json]: I hadn't either, but it's what React Static recommends and all their examples use it, so trying it out 😄

**Pull request merged**